pkg/cli/gh_gc_branch.go

```diff
@@ -53,6 +50,15 @@ wolfictl gh gc branch https://github.com/wolfi-dev/os --match "wolfictl-"
 	return cmd
 }
 
+type ghTokenSource struct{}
+
+func (ghTokenSource) Token() (*oauth2.Token, error) {
+	if tok, _ := ghauth.TokenForHost("github.com"); tok != "" {
+		return &oauth2.Token{AccessToken: tok}, nil
+	}
```

Comment:
should we have a fall back of checking if the `GITHUB_TOKEN` env var is set so that we don't break existing approaches that use the env var approach?

Reply:
`TokenForHost` checks the env var first and falls back.